### PR TITLE
Node.js Security Project Announcement

### PR DESCRIPTION
**Pull request by ZibbyKeaton**
Just looking for someone to help me get this published to the site. It might need some edits as I am not the best with this stuff. Thank you!

### CODE REVIEW

**Comment by MylesBorins** [comment]
@ZibbyKeaton thanks for putting this together, can I have your permission to push changes to your branch? There are some minor layout edits that need to be done

(Permission isn't a technical thing, just want you to say it is ok 😎)

**Comment by ZibbyKeaton** [comment]
@TheAlphaNerd yes please!

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by MylesBorins** [comment]
I've gone ahead and pushed some minor style changes, it should be good to go. I'd like it if one other person could put their eyes on it before landing.

@ZibbyKeaton having the date set to 11/30 is going to cause it to not be the most recent most, are you ok with that?

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (MylesBorins) [inline locale/en/blog/announcements/nodejssecurityproject.md]: should we make the publication date the current date?

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (MylesBorins) [inline locale/en/blog/announcements/nodejssecurityproject.md]: missed cleaning this dead link

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (MylesBorins) [inline locale/en/blog/announcements/nodejssecurityproject.md]: the carat in `^lift` should likely be in the link.

**Review by MylesBorins — APPROVED** [review]

**Comment by ZibbyKeaton** [comment]
@TheAlphaNerd that's totally fine.

**Comment by MylesBorins** [comment]
So I've gone ahead and made some more changes to the page, you can see notes above regarding what changes they were.

* remove dead links
* fix un-ordered list
* change publish date
* fix link for `^lift`

**Comment by hackygolucky** [comment]
LGTM!

**Review by hackygolucky — APPROVED** [review]

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (sam-github) [inline locale/en/blog/announcements/nodejssecurityproject.md]: Is this different from the "Security Working Group"? It sounds like a third group - the first being the group monitoring security vulnerabilities privately reported against node and the associated github.com/nodejs/security repo, the second being github.com/nodejs/security-wg and group forming there, and this is a third?

Comment (ZibbyKeaton) [reply locale/en/blog/announcements/nodejssecurityproject.md]: Yes, it's different from the Security Working group. The Node.js Security Project was recently run by a company called ^Lift, but is migrating over to the Node.js Foundation.

Comment (MylesBorins) [reply locale/en/blog/announcements/nodejssecurityproject.md]: I think the challenge with the language here is including `begin validating vulnerability disclosures`

This may be true for vulnerabilities in NPM packages... but is not at all the case for vulnerabilities in node, which will still go through the current security wg and security@nodejs.org

The messaging here is critical, as we embargo vulnerabilities before they are properly patched and reported, sending people to the wrong place can be a serious issue.

**Review by MylesBorins — CHANGES_REQUESTED** [review]
I'm holding off on my approval until we can clarify language regarding security wg

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (mikeal) [reply locale/en/blog/announcements/nodejssecurityproject.md]: So, here's the problem.

This press release actually already went out. In fact, it went out a few weeks ago, and we want the language to match what was sent to journalists when it was announced.

*When it was drafted* the language was accurate to what we *thought* things would look like. I'm not surprised that once we've dug in things have changed.

What we should do here is back date this announcement to when it went out a few weeks ago. Once the security working group has some documentation produced (they are still in the initial steps of getting the WG setup) we can do another post either on the blog or on Medium about the structure and processes.

**Comment by mikeal** [comment]
@TheAlphaNerd see my inline comments.

**Comment by ZibbyKeaton** [comment]
Checking in to see if we've come to a resolution here? @sam-github @TheAlphaNerd. As Mikeal was saying, this press release went out on November 30th. I wasn't able to post it at the site at that time, but this was reviewed and approved by Mikeal and Adam Baldwin. It sounds like a bit of this has changed, but I don't think it makes that much sense to retro-activly go back and change the release. Maybe it would make sense to add a link in this that states, changes have been made and are continuing to be made to this, please see and review the conversation here and link back to where these changes and discussions have been made? Would that be a good alternative to this?

**Comment by sam-github** [comment]
I've no objections to anything here, @mikeal answered my question, and if he's good with this press release, I am.

File: locale/en/blog/announcements/nodejssecurityproject.md
Comment (Fishrock123) [reply locale/en/blog/announcements/nodejssecurityproject.md]: I don't think that is useful. Let's just update the language and publish that, how is that in any way problematic?

Comment (hackygolucky) [reply locale/en/blog/announcements/nodejssecurityproject.md]: The press releases are like meeting minutes and a bit of a time capsule. The documents are being uploaded to this section of the website as much for news delivery as they are an archive of what was disseminated to news groups as the official press release. Those news groups, if publishing the press release, don't modify it.

**Review by MylesBorins — APPROVED** [review]

**Comment by MylesBorins** [comment]
I've gone ahead and approved this press release. We need to document what was originally sent out.

**Comment by MylesBorins** [comment]
landed in 7664c7e3dc

**Comment by ZibbyKeaton** [comment]
@TheAlphaNerd thank you!

**Pull request closed without merging**
